popsycle/synthetic.py

```diff
@@ -214,9 +229,15 @@ def perform_pop_syn(ebf_file, output_root, iso_dir,
     NS_kick_speed : float 
         Kick speed of NS (in km/s)
 
-    set_random_seed : bool
-        Forces PyPopStar to fix the random seed to 42,
-        enforcing identical output.
+    overwrite : bool
+        If set to True, overwrites output files. If set to False, exists the
+        function if output files are already on disk.
+        Default is False.
+
+    seed : int
+        If set to non-zero, removes all random sampling and forces identical
```

Comment:
This comment isn't quite accurate. Suggest changing to:

If set to non-zero, then all random sampling will be seeded with the specified seed.

Reply:
Changed